CHANGELOG.md

```diff
@@ -4,6 +4,9 @@ All notable changes to [Earthly](https://github.com/earthly/earthly) will be doc
 
 ## Unreleased
 
+### Added
+- New Github Actions Workflow commands integration `--github-annotations` flag or GITHUB_ACTIONS=true env. See [#4047](https://github.com/earthly/earthly/pull/4047)
```

Comment:
nit: we tend to only have the issue at the end of changelog entries, the "See" is implied by the link.
```suggestion
- New Github Actions Workflow commands integration `--github-annotations` flag or GITHUB_ACTIONS=true env. [#4047](https://github.com/earthly/earthly/pull/4047)
```

Reply:
Done